packages/wpcom-checkout/package.json

```diff
@@ -37,6 +37,7 @@
 	"dependencies": {
 		"@automattic/calypso-config": "workspace:^",
 		"@automattic/calypso-products": "workspace:^",
+		"@automattic/calypso-razorpay": "^0.0.1",
```

Comment:
Is there a reason why this is not `workspace` too?

Reply:
My understanding is this is needed because the calypso-razorpay package is not yet published; the [dev docs](https://github.com/Automattic/wp-calypso/blob/trunk/docs/dependency-management.md#unpublished-package-as-a-dependency) talk about this. I'm not 100% sure on what the difference is though.